js/indicia.datacomponents/jquery.idc.filterSummary.js

```diff
@@ -193,6 +193,47 @@
         });
       }
 
+      // es-filter-param controls
+      if ($('.es-filter-param').length) {
+        $.each($('.es-filter-param'), function eachParam() {
+          var val = $(this).val();
+          // Skip if no value.
+          if (val === null || val.trim() === '') {
+            return;
+          }
+          // Skip if unchecked checkbox
+          if ($(this).is(':checkbox') && !$(this).is(':checked')) {
+            return;
+          }
+   
+          var esBoolClause = indiciaFns.getDataValueFromInput(this, 'data-es-bool-clause');
+          var esField = indiciaFns.getDataValueFromInput(this, 'data-es-field');
+          var esQueryType = indiciaFns.getDataValueFromInput(this, 'data-es-query-type');
+          var esQuery = indiciaFns.getDataValueFromInput(this, 'data-es-query');
+          var esNested = indiciaFns.getDataValueFromInput(this, 'data-es-nested');
+
+          html = addHtml(html, '<div><b>ES query: </b>');
```

Comment:
Is the abbreviation ES something the end-user will understand?

Reply:
No I agree. I've changed 'ES query' to 'Form filter'.